api4/command_test.go

```diff
@@ -107,51 +106,51 @@ func TestUpdateCommand(t *testing.T) {
 		Token:     "tokenchange",
 	}
 
-	rcmd, resp := Client.UpdateCommand(cmd2)
-	CheckNoError(t, resp)
+	th.TestForSystemAdminAndLocal(t, func(t *testing.T, client *model.Client4) {
```

Comment:
As these tests where originally running for `th.Client`, shouldn't this be:

```suggestion
	th.TestForAllClients(t, func(t *testing.T, client *model.Client4) {
```

Reply:
@mgdelacroix  original client was th.SystemAdminClient so this should be TestForSystemAdminAndLocal

Reply:
You're totally right! :facepalm: